cli/server_request_command.go

```diff
@@ -138,6 +146,45 @@ func (c *SrvRequestCmd) healthz(_ *fisk.ParseContext) error {
 	return nil
 }
 
+func (c *SrvRequestCmd) profilez(_ *fisk.ParseContext) error {
+	nc, _, err := prepareHelper("", natsOpts()...)
+	if err != nil {
+		return err
+	}
+
+	opts := server.ProfilezEventOptions{
+		ProfilezOptions: server.ProfilezOptions{
+			Name:  c.profileName,
+			Debug: c.profileDebug,
+		},
+		EventFilterOptions: c.reqFilter(),
+	}
+
+	res, err := c.doReq("PROFILEZ", &opts, nc)
+	if err != nil {
+		return err
+	}
+
+	var response struct {
+		Resp server.ProfilezStatus `json:"data"`
+	}
+
+	var b []byte
+	for _, r := range res {
+		b = append(b, r...)
+	}
+	if err := json.Unmarshal(b, &response); err != nil {
+		return fmt.Errorf("no response")
+	}
+
+	if response.Resp.Error != "" {
+		return fmt.Errorf(response.Resp.Error)
+	}
+	fmt.Print(string(response.Resp.Profile))
```

Comment:
probably need a new line here

Reply:
Are all the profiles text? nothing thats the pprof gzip stuff?

Reply:
Since the response is just `[]byte` it did do the right thing for gzipped profiles, but to make it clearer that it might be binary, I've replaced it by writing directly to `os.Stdout` using a buffer.